plugins/sql_extract/src/sql_extract/component.py

```
@@ -0,0 +1,264 @@
+                echo=False,  # Установите True для отладки SQL запросов
+            )
+
+            # Создаем фабрику сессий
+            self._session_factory = async_sessionmaker(
+                self._engine, expire_on_commit=False
+            )
+
+            logger.debug(f"SQLAlchemy движок создан для {uri}")
+
+        except Exception as e:
+            msg = f"Не удалось создать SQLAlchemy движок: {e}"
```

Ruff (0.12.2)

String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)